plugin-api/src/main/java/net/clgd/ccemux/config/JsonAdapter.java

```diff
@@ -121,10 +121,9 @@ private void fromJson(Group group, JsonElement element) {
 				if (configEntry.get() instanceof Group) {
 					fromJson((Group) configEntry.get(), jsonEntry.getValue());
 				} else if (configEntry.get() instanceof ConfigProperty<?>) {
-					ConfigProperty property = (ConfigProperty<?>) configEntry.get();
```

Comment:
Does this need to be migrated as well? Seeing as we're going for a grand-unified-config-file, I'm not sure I see the need for exposing it?

Reply:
Probably not, I had just moved the entire `config` package.